ONSdigital/eq-author
# Fix for #59 - adding database transaction

### What is the context of this PR?
Fixes #59 . Wrapping the schema api in a database transaction, this means it rolls back if an error occurs rather than leaving a partially created questionnaire in the database. 

Also temporarily create a survey if one doesn't exist (this will be removed as part of the create survey story(

Finally removing an incorrect migration in the old qBuilder folder

### How to review 
1. Launch the author application
2. Check you can add a questionnaire


### Who worked on the PR
@warrenbailey
